tests/ejection_test/ejection_test.ino

```diff
@@ -0,0 +1,17 @@
+// TODO(1): adapt to actual pin
+int FIRE_PIN = LED_BUILTIN;
+
+void setup() {
+  Serial.begin(9600);
+
+  pinMode(FIRE_PIN, OUTPUT);
+
+  // TODO(2): is this countdown enough?
+  // wait for 20 sec
+  delay(20000);
```

Comment:
```suggestion
  // TODO(2): is this countdown enough?
  // wait for 120 sec
  delay(120000);
```
I'd say not enough. Let's give us time to get camera ready and tape the nose cone back into place with the battery before it fires. (also when we rush to move we might damage the board)